lib/src/disposable/disposable.dart

```diff
@@ -427,6 +428,28 @@ class Disposable implements _Disposable, DisposableManagerV3 {
     return null;
   }
 
+  void subscribeToDocumentEvent(String event, EventListener callback,
```

Comment:
It looks like all the managing methods on `Disposable` are exposed that way, so probably.

We have a separate ticket for adding a `subscribeToStream` method to `Disposable`, so maybe we should wait until that's done and add all of these new methods to `DisposableManager` at once, so we only have to deprecate once? @georgelesica-wf what do you think?